src/cmdlib.sh

```diff
@@ -276,6 +276,16 @@ baseurl=file://${workdir}/overrides/rpm
 gpgcheck=0
 EOF
     fi
+    rootfs_overlay="${overridesdir}/rootfs"
+    if [ -d "${rootfs_overlay}" ]; then
+        echo "Committing ${rootfs_overlay}"
+        sudo ostree commit --repo="${workdir}/tmp/repo" --tree=dir="${rootfs_overlay}" -b cosa-bin-overlay \
+          --owner-uid 0 --owner-gid 0 --no-xattrs --no-bindings --parent=none
+          cat >> "${override_manifest}" << EOF
+ostree-override-layers:
+  - cosa-bin-overlay
+EOF
+     fi
```

Comment:
Super minor: looks like indentation is off here?

Reply:
:no_entry_sign: ` `